### DISCUSSION TITLE
StokesFlow.run() show "Exception: Your network is clustered, making Ax = b ill-conditioned"

### DISCUSSION THREAD

**Discussion by eromanga233** [PMEAL/OpenPNM#2814] [Q&A]
Dear professor:
    When calculating the absolute permeability of digital cores,  I use `pn = op.io.network_from_porespy(net.network)` to construct a network in openpnm. However, it always goes as 'Exception: Your network is clustered, making Ax = b ill-conditioned'. I am wondering what makes this error happen and how can I fix it ?

**Comment by jgostick** [comment]
This issue comes up a lot so I wrote a detailed example that should help: https://openpnm.org/examples/reference/networks/managing_clustered_networks.html
